integration-cli/docker_cli_restart_test.go

```diff
@@ -74,15 +74,15 @@ func (s *DockerSuite) TestRestartWithVolumes(c *check.C) {
 }
 
 func (s *DockerSuite) TestRestartPolicyNO(c *check.C) {
-	out, _ := dockerCmd(c, "run", "-d", "--restart=no", "busybox", "false")
+	out, _ := dockerCmd(c, "create", "--restart=no", "busybox")
```

Comment:
These tests only did an inspect on the container-config, so no "run" was needed. I changed to "create" to save some time, but happy to change back if you don't agree with changing this